PCbuild/readme.txt

```diff
@@ -52,6 +52,42 @@ Release
     settings, though without PGO.
 
 
+Building Python using Clang/LLVM
+--------------------------------
+
+See https://learn.microsoft.com/en-us/cpp/build/clang-support-msbuild?view=msvc-170
+how to install and use clang-cl bundled with Microsoft Visual Studio.
+You can use the IDE to switch to clang-cl for local development,
+but because this alters the *.vcxproj files, the recommended way is
+to use build.bat:
+
+build.bat "/p:PlatformToolset=ClangCL" "/p:PreferredToolArchitecture=x64"
+
+All other build.bat options remain to work as with MSVC, so this
+will create a 64bit release binary. PreferredToolArchitecture is needed,
+because msbuild by default selects the 32bit architecture of the toolset,
+which uses -m32 as the default target architecture.
```

Comment:
```suggestion
All other build.bat options continue to work as with MSVC, so this
will create a 64bit release binary. PreferredToolArchitecture is needed,
because msbuild by default selects the 32bit architecture of the toolset,
which uses -m32 as the default target architecture.
```